laravel/resources/views/schedules/resident/milestone.blade.php

```diff
@@ -35,7 +35,7 @@
 			<div id="anesthesiologist_preference">
 				<label>Anesthesiologist Preference:</label>
 				<br>
-				<select class="PreferenceSelector">
+				<select name = "prefAnest1" id = "prefAnest1" class="PreferenceSelector">
```

Comment:
formatting is off. no space around `=`.

also, i'd prefer to use `pref-anest1`. HTML/CSS best practice is to use dashes to separate words in ids, classes, names, etc.

hopefully it goes without saying, but this change should happen for the other selects too.